tests/test_pallet.py

```diff
@@ -345,3 +345,38 @@ def test_processing_time(self):
         self.assertEqual(round(pallet.total_processing_time), 5)
         self.assertEqual(round(pallet.processing_times[first]), 2)
         self.assertEqual(round(pallet.processing_times[second]), 3)
+
+
+class TestPalletAddPackingSlip(unittest.TestCase):
+    def test_successful_crate(self):
+        slip = {
+            "name": "c:\\forklift\\warehouse\\warehouse\\sgid\\AGOLPallet.py:AGOLPallet",
+            "success": True,
+            "is_ready_to_ship": True,
+            "requires_processing": True,
+            "message": "",
+            "crates": [
+                {
+                    "name": "fc5",
+                    "result": "Data updated successfully.",
+                    "crate_message": "",
+                    "message_level": "",
+                    "source": "c:\\users\\agrc-arcgis\\appdata\\local\\esri\\conda\\envs\\forklift\\lib\\site-packages\\forklift\\..\\forklift-garage\\sgid10.sde\\SGID10.BOUNDARIES.ZipCodes",
```

Comment:
%localappdata% might be a better path to appdata\local

Reply:
i don't think this works actually so... nevermind? Unless you want to research another solution

Reply:
The path doesn't really matter in this test.